core/templates/domain/suggestion/suggestion.model.ts

```diff
@@ -62,10 +58,10 @@ export class Suggestion {
       stateName: string, newValue: SuggestionChangeValue,
       oldValue: SuggestionChangeValue, lastUpdatedMsecs: number) {
     this.suggestionType = suggestionType;
-    this.suggestionId = suggestionId;
     this.threadId = threadId;
     this.targetType = targetType;
     this.targetId = targetId;
+    this.suggestionId = suggestionId;
```

Comment:
Why you moved this? I'm fine with that but don't see a reason for that.

Reply:
Oh I must have moved it by mistake. Fixed it!